modules/insights-operator-enable-obfuscation.adoc

```diff
@@ -33,3 +36,13 @@ For cluster base domains, obfuscation changes the base domain to a hardcoded sub
 . Select the `openshift-insights` project.
 . Find the `insights-operator` pod.
 . To restart the `insights-operator` pod, click the *Options* menu {kebab}, and then click *Delete Pod*.
+
+.Verification Steps
+
+. Navigate to *Workloads* -> *Secrets*.
+. Select the *openshift-insights* project.
+. Search for the *obfuscation-translation-table* secret using the *Search by name* field.
+
+If the `obfuscation-translation-table` secret exists, then obfuscation is enabled and working. 
+
+Alternatively, you can inspect `/insights-operator/gathers.json` in your xref:../../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc#insights-operator-showing-data-collected-from-the-cluster_showing-data-collected-by-remote-health-monitoring[Insights Operator archive] for the value `"is_global_obfuscation_enabled": true`.
```

Comment:
We're not allowed to add xrefs to modules. If you'd like to have an xref, you'll want to add an ".Additional resources" in the assembly right after the module is included, and add the xref there.

Reply:
can I just make this a regular link?